src/promptflow-evals/promptflow/evals/evaluate/_utils.py

```diff
@@ -64,66 +70,107 @@ def _write_properties_to_run_history(properties: dict) -> None:
         LOGGER.error("Fail writing properties '%s' to run history: %s", properties, e)
 
 
-def _azure_pf_client_and_triad(trace_destination):
-    from promptflow.azure._cli._utils import _get_azure_pf_client
+def _get_credential(force_cli: bool = False) -> TokenCredential:
+    """
+    Return the credential found in the system.
+
+    :param force_cli: Force using CLI credentials.
+    :type force_cli: bool
+    :return: The token credential.
+    """
+    if force_cli:
+        return AzureCliCredential()
+    if os.environ.get("AZUREML_OBO_ENABLED"):
+        return AzureMLOnBehalfOfCredential()
+    if os.environ.get("DEFAULT_IDENTITY_CLIENT_ID"):
+        return ManagedIdentityCredential(
+            client_id=os.environ.get("DEFAULT_IDENTITY_CLIENT_ID"))
+    try:
+        return DefaultAzureCredential()
+    except BaseException:
+        return AzureCliCredential()
+
+
+def _get_ml_client(trace_destination: str, **kwargs) -> Tuple[Any, AzureMLWorkspaceTriad]:
+    """
+    Return the MLclient and workspace triad.
+
+    :param trace_destination: The URI, containing subscription ID,
+                              workspace and resource group name.
+    :type trace_destination: str
+    :return: The tuple with the ML client and workspace triad.
+    """
+    try:
+        from azure.ai.ml import MLClient
+    except (ImportError, ModuleNotFoundError):
+        LOGGER.error('Unable to import promptflow-azure, the run will not be logged to azure. '

```

Comment:
Since we already added the check at PFClient initialization, the check here is probably not necessary.

Reply:
Fixed